api/app/main.py

```diff
@@ -1,21 +1,43 @@
+import os
+from dotenv import load_dotenv
+
 import asyncpg
 import logging
 from fastapi import FastAPI, HTTPException, Body
-from web_scraper import WebScraper
+from fastapi.middleware.cors import CORSMiddleware
 
+from web_scraper import WebScraper
 
+# Setup logging
 logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
 logger = logging.getLogger(__name__)
 
 # Create an instance of FastAPI
 app = FastAPI()
 
+# Load environment variables
+load_dotenv()
+CORS_ALLOWED_ORIGINS = os.environ["CORS_ALLOWED_ORIGINS"]
+POSTGRES_USER = os.environ["POSTGRES_USER"]
+POSTGRES_PASSWORD = os.environ["POSTGRES_PASSWORD"]
+POSTGRES_DB = os.environ["POSTGRES_DB"]
+POSTGRES_HOST = os.environ["POSTGRES_HOST"]
+
+# Setup CORS
+app.add_middleware(
```

Comment:
please describe in 1-2 sentences why we are doing this and how tight it is necessary.

Reply:
Here we define the service name ("db") of the PostgreSQL database in the docker-compose.yml file. This "db" service runs the PostgreSQL server. I will leave this hardcoded to simplify.